flexmeasures/data/services/users.py

```diff
@@ -40,8 +41,10 @@ def get_users(
     role_name: str | None = None,
```

Comment:
I believe that you are not using this utility function after all. In fact, with this PR it seems no place is using it and it can be deleted entirely.